### PR TITLE
functions-revamp and whatsnew

### PR DESCRIPTION
**Pull request by monishas9011**

### CODE REVIEW

File: docs/cookbooks/studio/usecases-functions.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: Function use cases
+sidebar_label: Function use cases
+---
+
+The following are some of the use cases of the **Functions** section in **Studio**.
+
+## Display  weather in your bot in a caraousel node
```
Comment (Architha-j) [inline docs/cookbooks/studio/usecases-functions.md:8]: Function (code) to Fetch weather and display it in a carousel node

File: docs/cookbooks/studio/usecases-functions.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+4. Pass that variable in the text node to display the result to the end user.
+
+![](https://i.imgur.com/2cCX1By.png)
+
+**Result:**
+
+<img src="https://i.imgur.com/fioBFmp.png" alt="drawing" width="60%"/>
+
+
+## Codes to perform sentence autocompletion and decode BASE64
```
Comment (Architha-j) [inline docs/cookbooks/studio/usecases-functions.md:95]: add this as separate headings. 

## Function (code) to enable autocompletion
## Function (code) to decode BASE64

File: docs/cookbooks/studio/usecases-functions.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+![](https://i.imgur.com/Mlh4Xzv.png)
+
+:::note
+Instead of including the Function node, you can also pass the function in the **Parse API response** field.
+![](https://i.imgur.com/eIReCNg.png)
+:::
+
+
+## Calculate 45 days before the present date
```
Comment (Architha-j) [inline docs/cookbooks/studio/usecases-functions.md:37]: ## Function (code) to manipulate dates

Add example as 45 days before the present date

Explain date object and say, you can write functions to calculate based on these fields ex. timestamp, year, date etc.

File: docs/platform_concepts/studio/build/code.md [hunk 4]
```diff
@@ -4,303 +4,135 @@ sidebar_label : Function (code)
...
 
-:::note
-To work around this section, you will require minimal coding skills.
-:::
+## Create a function
 
+This section covers the format for writing custom logic, the available arguments for data retrieval, and how to create functions using this information.
 
-The Function section is used to add function code. It is available in **Studio** -> **Build** -> **Function**.
+### Format of functions
```
Comment (Architha-j) [inline docs/platform_concepts/studio/build/code.md:15]: Function (syntax) of a function

**Review by Architha-j — COMMENTED** [review]
Added a few comments please have a look

File: docs/cookbooks/studio/usecases-functions.md (hunk 1 above)
Comment (monishas9011) [reply docs/cookbooks/studio/usecases-functions.md:8]: Doesn't suit the steps as we are not giving the function(code) here to display the weather, we are just giving them the steps to do so. The current heading works for this.

File: docs/cookbooks/studio/usecases-functions.md (hunk 3 above)
Comment (monishas9011) [reply docs/cookbooks/studio/usecases-functions.md:37]: seems like a stretch, we have taken a specific example to explain here. Better to stick only to that. Use case guides are intended towards a particular use case

File: docs/platform_concepts/studio/build/code.md (hunk 4 above)
Comment (monishas9011) [reply docs/platform_concepts/studio/build/code.md:15]: Format also works here.

File: docs/cookbooks/studio/usecases-functions.md (hunk 2 above)
Comment (monishas9011) [reply docs/cookbooks/studio/usecases-functions.md:95]: separated the headings

**Pull request merged**